src/pages/trainingVideos.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { MediaCard } from "../components/UI/MediaCard";
+import { RegisterButton } from "./../components/WebinarPageContent/RegisterButton";
+
+export const TrainingVideosPage = () => (
+  <>
+    <Helmet>
+      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+      <meta property="og:description" content="Add Tradetrust description" />
```

Comment:
Lol want to fix this? Can help to give it a good page title too

Reply:
yup sure, fixed the other pages too